test/base-install-node/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM test/nodejs
+COPY . /app/
+RUN npm install --unsafe-perm || \
+  ((if [ -f npm-debug.log ]; then \
+      cat npm-debug.log; \
+    fi) && false)
```

Comment:
`&& false` helps us print the npm-debug.log but preserves the exit code (specifically that `npm install` failed).